pkgbuild/packagesbuild.sh

```diff
@@ -162,7 +162,7 @@ cp -R "${INPUT_DIRECTORY}" "${DIRECTORY}"
 if [ ! -z "$SIGN_OPTION" ]; then
     xattr -cr .
     security unlock-keychain -p `cat ~/.password` login.keychain-db
-    /usr/bin/codesign --verbose=4 --deep --force -s "Developer ID Application: London Jamocha Community CIC" ${DIRECTORY}
+    /usr/bin/codesign --verbose=4 --deep --force -s - ${DIRECTORY}
```

Comment:
is the `-` correct?

Reply:
it's a template, it allows other users to add their cert (see the comment a few lines up)